static/app/views/insights/types.tsx

```diff
@@ -499,6 +499,7 @@ type MetricsNumberFields =
   | MetricsFields.INP
   | MetricsFields.CLS
   | MetricsFields.TTFB
+  | MetricsFields.TTFB
```

Comment:
Accidental duplication?

Reply:
nice catch, i think it was a merge conflict actually. There's a test hanging here too that i need to solve, will put this into draft for now!